dist/obsworker

```diff
@@ -352,8 +335,8 @@ case "$1" in
             NUM=`ls -d /sys/devices/system/cpu/cpu[0-9]* | wc -l`
         fi
         if [ "--zvm" == "$vmopt" ]; then
-            check_vmcp || rc_status -v
-            create_initrd $OBS_VM_KERNEL $OBS_VM_INITRD || rc_status -v
+            check_vmcp
+            create_initrd $OBS_VM_KERNEL $OBS_VM_INITRD
```

Comment:
you're loosing the state here, should probably just be

check_vmcp || exit 1
create_initrd .... || exit 1

or something else to fail if intermediate steps have failed

Reply:
Done. I substituted these lines as you proposed and added a comment in the affected commit.

Please, re-review it again.